### PR TITLE
Remove the freeze graph view feature

### PR DESCRIPTION
**Pull request by arcturus2**
### Description of the Change

The pin/unpin feature should be able to replace the functionality of "Freeze graph". All capability related to freeze graph view have been removed effectively as if it was turned off.

Though this change should not make much of a difference given freeze graph is off by default, there can be unintended consequences so I'm OK if this feature does not go into the v2.3 release.

However if there is sufficient testing and it looks fine then happy for this to be included.

### Benefits

Reduce tech debt

### Possible Drawbacks

There is a small possibility this can impact the existing behaviour of the pin/unpin though I doubt it. Testing will help confirm this.

### CODE REVIEW

File: CoreTesting/src/au/gov/asd/tac/constellation/testing/construction/CompleteGraphBuilderPlugin.java [hunk 1]
```diff
@@ -272,22 +270,18 @@ public void edit(final GraphWriteMethods graph, final PluginInteraction interact
...
+                PluginExecutor.startWith(ArrangementPluginRegistry.TREES)
+                        .followedBy(InteractiveGraphPluginRegistry.RESET_VIEW).executeNow(graph);
+            } else {
+                // Do a grid layout.
+                PluginExecutor.startWith(ArrangementPluginRegistry.GRID_COMPOSITE)
+                        .followedBy(InteractiveGraphPluginRegistry.RESET_VIEW).executeNow(graph);
             }
-        } else {
-            PluginExecution.withPlugin(InteractiveGraphPluginRegistry.RESET_VIEW).executeNow(graph);
+        } catch (PluginException ex) {
```
Comment (antares1470) [inline CoreTesting/src/au/gov/asd/tac/constellation/testing/construction/CompleteGraphBuilderPlugin.java:283]: I know this was just a re-format but would be good if you could make this final

File: CoreTesting/src/au/gov/asd/tac/constellation/testing/construction/PreferentialAttachmentGraphBuilderPlugin.java [hunk 2]
```diff
@@ -301,22 +299,18 @@ public void edit(final GraphWriteMethods graph, final PluginInteraction interact
...
+                PluginExecutor.startWith(ArrangementPluginRegistry.TREES)
+                        .followedBy(InteractiveGraphPluginRegistry.RESET_VIEW).executeNow(graph);
+            } else {
+                // Do a grid layout.
+                PluginExecutor.startWith(ArrangementPluginRegistry.GRID_COMPOSITE)
+                        .followedBy(InteractiveGraphPluginRegistry.RESET_VIEW).executeNow(graph);
             }
-        } else {
-            PluginExecution.withPlugin(InteractiveGraphPluginRegistry.RESET_VIEW).executeNow(graph);
+        } catch (PluginException ex) {
```
Comment (antares1470) [inline CoreTesting/src/au/gov/asd/tac/constellation/testing/construction/PreferentialAttachmentGraphBuilderPlugin.java:312]: same here

File: CoreTesting/src/au/gov/asd/tac/constellation/testing/construction/CompleteGraphBuilderPlugin.java (hunk 1 above)
Comment (antares1470) [inline CoreTesting/src/au/gov/asd/tac/constellation/testing/construction/CompleteGraphBuilderPlugin.java:283]: I know this was just a re-format but would be good if you could make this final

File: CoreTesting/src/au/gov/asd/tac/constellation/testing/construction/PreferentialAttachmentGraphBuilderPlugin.java (hunk 2 above)
Comment (antares1470) [inline CoreTesting/src/au/gov/asd/tac/constellation/testing/construction/PreferentialAttachmentGraphBuilderPlugin.java:312]: same here

File: CoreTesting/src/au/gov/asd/tac/constellation/testing/construction/SmallWorldGraphBuilderPlugin.java [hunk 3]
```diff
@@ -399,22 +397,18 @@ public void edit(final GraphWriteMethods graph, final PluginInteraction interact
...
+                PluginExecutor.startWith(ArrangementPluginRegistry.TREES)
+                        .followedBy(InteractiveGraphPluginRegistry.RESET_VIEW).executeNow(graph);
+            } else {
+                // Do a grid layout.
+                PluginExecutor.startWith(ArrangementPluginRegistry.GRID_COMPOSITE)
+                        .followedBy(InteractiveGraphPluginRegistry.RESET_VIEW).executeNow(graph);
             }
-        } else {
-            PluginExecution.withPlugin(InteractiveGraphPluginRegistry.RESET_VIEW).executeNow(graph);
+        } catch (PluginException ex) {
```
Comment (antares1470) [inline CoreTesting/src/au/gov/asd/tac/constellation/testing/construction/SmallWorldGraphBuilderPlugin.java:410]: and here

File: CoreTesting/src/au/gov/asd/tac/constellation/testing/construction/CompleteGraphBuilderPlugin.java (hunk 1 above)
Comment (arcturus2) [reply CoreTesting/src/au/gov/asd/tac/constellation/testing/construction/CompleteGraphBuilderPlugin.java]: Added a final

File: CoreTesting/src/au/gov/asd/tac/constellation/testing/construction/PreferentialAttachmentGraphBuilderPlugin.java (hunk 2 above)
Comment (arcturus2) [reply CoreTesting/src/au/gov/asd/tac/constellation/testing/construction/PreferentialAttachmentGraphBuilderPlugin.java]: Added a final

File: CoreTesting/src/au/gov/asd/tac/constellation/testing/construction/SmallWorldGraphBuilderPlugin.java (hunk 3 above)
Comment (arcturus2) [reply CoreTesting/src/au/gov/asd/tac/constellation/testing/construction/SmallWorldGraphBuilderPlugin.java]: Added a final

**Comment by arcturus2** [comment]
@antares1470 added finals and fixed the merge conflicts. Ready for review.

**Review by antares1470 — CHANGES_REQUESTED** [review]
Seems to all be working alright. Maybe a Changelog entry to note this removal would be useful (maybe a whatsnew as well just in case people were using freeze graph)

**Comment by arcturus2** [comment]
> 
> 
> Seems to all be working alright. Maybe a Changelog entry to note this removal would be useful (maybe a whatsnew as well just in case people were using freeze graph)

No worries @antares1470 ready for another review.

**Review by antares1470 — APPROVED** [review]

**Comment by GammaVel** [comment]
Hi @arcturus2 will the help guide be updated as part of this change?

**Comment by arcturus2** [comment]
@GammaVel sorry about the delayed response but letting you know that the doco was also updated.

**Pull request merged**